toqito/state_props/tests/test_is_abs_ppt.py

```diff
@@ -0,0 +1,64 @@
+"""Unit tests for the is_abs_ppt function in toqito.state_props."""
+
+import numpy as np
+import pytest
+
+from toqito.rand import random_psd_operator
+from toqito.state_props import is_abs_ppt
+from toqito.states import max_mixed
+
+
+@pytest.mark.parametrize(
+    "dims, n",
+    [([2, 2], 4), ([2, 3], 6), ([3, 3], 9)]
+)
+def test_maximally_mixed_states(dims, n):
+    """Test that maximally mixed states are absolutely PPT for supported dimensions."""
+    rho = max_mixed(n)
+    assert is_abs_ppt(rho, dims) is True
+
+
+@pytest.mark.parametrize("dims", [([2, 2]), ([2, 3]), ([3, 3])])
+def test_random_psd_not_absolutely_ppt(dims):
+    """Test that random PSD states are not always absolutely PPT."""
+    np.random.seed(42)
+    n = dims[0] * dims[1]
+    for _ in range(3):
+        rho = random_psd_operator(n)
+        rho = rho / np.trace(rho)
+        res = is_abs_ppt(rho, dims)
+        assert res in [True, False, -1]
```

Comment:
Not the way to go about verifying this. The result is always going to be in this list. Add an additional var for the `expected_result` and then verify it.